core/domain/prod_validators.py

```diff
@@ -2873,182 +2633,6 @@ def _get_model_domain_object_instance(cls, item):
             .create_community_contribution_stats_from_model(item)
         )
 
-    @classmethod
```

Comment:
Why did we remove these functions (`_validate_translation_reviewer_counts`, `_validate_translation_suggestion_counts`, etc.)? Maybe we should put them back.